include/fw_info.h

```diff
@@ -311,6 +375,16 @@ const struct fw_info_abi *fw_info_abi_get(u32_t id, u32_t index);
 const struct fw_info_abi *fw_info_abi_find(u32_t id, u32_t flags,
 					u32_t min_version, u32_t max_version);
 
+
+/**Invalidate this image by setting the @c valid value to 0x0.
```

Comment:
Possible difficulty of matching "image" to content inside fw_info (the valid flag)
since the param[in] fw_info is talking about invalidating the info structure

Suggestion:

Function that invalidates an image by manipulating fw_info

@details Setting the @c valid value to 0x0 in fw_info, invalidates the image it represents

Reply:
Changed to

    /**Invalidate an image by manipulating its fw_info.
     * 
     * @details Invalidation happens by setting the @c valid value to 0x0.